.github/GIT_DEV_GUIDELINE.md

```diff
@@ -0,0 +1,30 @@
+# GIT development guideline
+## Commandments:
+### Branch
+- Create a new branch for every task / small feature / bug fix from `master`
+- Branch prefix:
+  - Feature: `feature/`
+  - Release: `release/`
+  - Hotfix: `hotfix/`
+  - Support: `support/`
+- Branch name should refer to issue number: e.g `feature/#9_referral_link`
+### Commit
+- Make small/atomic commits.
+- Do not commit vendor/compiled code.
+- Do not commit commented-out code — just delete it.
+- Explain the why, not the what, in your commit message:
```

Comment:
Cái này có thể reference thêm: How to write a git commit message của Chris Beams